<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>codeEditPage</class>
 <widget class="QWidget" name="codeEditPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1492</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="0" column="0" rowspan="4">
      <widget class="QWidget" name="codeEditorArea" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2" rowspan="4">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="StrongBodyLabel" name="selectedHeader">
           <property name="text">
            <string>已选择的独立变量表</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="TableWidget" name="selectedTabView">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>250</height>
            </size>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="StrongBodyLabel" name="dependentHeaders">
           <property name="text">
            <string>约束变量表</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ToolButton" name="dependentAdd">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="ToolButton" name="dependentDelete">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="TableWidget" name="dependentTabView">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="StrongBodyLabel" name="goalHeader">
           <property name="text">
            <string>待优化目标表</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ToolButton" name="goalAdd">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="ToolButton" name="goalDelete">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="TableWidget" name="goalTabView">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="PrimaryPushButton" name="startOptimizeButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
